ui.apps/webpack.config.js

```diff
@@ -26,44 +26,69 @@ const LIB = {
     NAVIGATION: 'apps/core/cif/components/structure/navigation/v1/navigation/clientlibs'
 };
 
-module.exports = {
-    entry: {
-        // Map of clientlib base paths and a corresponding array of JavaScript files that should be packed. We use the
-        // key to specify the target destination of the packed code and the glob module to generate a list of JavaScript
-        // files matching the given glob expression.
-        [LIB.COMMON]: ['@babel/polyfill', ...glob.sync(JCR_ROOT + LIB.COMMON + '/js/**/*.js')],
-        [LIB.MINICART]: glob.sync(JCR_ROOT + LIB.MINICART + '/js/**/*.js'),
-        [LIB.PRODUCT]: glob.sync(JCR_ROOT + LIB.PRODUCT + '/js/**/*.js'),
-        [LIB.PRODUCTCAROUSEL]: glob.sync(JCR_ROOT + LIB.PRODUCTCAROUSEL + '/js/**/*.js'),
-        [LIB.PRODUCTLIST]: glob.sync(JCR_ROOT + LIB.PRODUCTLIST + '/js/**/*.js'),
-        [LIB.HEADER]: glob.sync(JCR_ROOT + LIB.HEADER + '/js/**/*.js'),
-        [LIB.NAVIGATION]: glob.sync(JCR_ROOT + LIB.NAVIGATION + '/js/**/*.js')
-    },
-    output: {
-        path: path.resolve(__dirname, "src/main/content/jcr_root"),
-        // [name] will be replaced by the base path of the clientlib (key of the entry map).
-        filename: './[name]/dist/index.js'
-    },
-    module: {
-        rules: [
-            // Transpile .js files with babel. Babel will by default pick up the browserslist definition in the 
-            // package.json file.
-            {
-                test: /\.js$/,
-                exclude: /(node_modules|bower_components)/,
-                use: {
-                    loader: 'babel-loader',
-                    options: {
-                        presets: ['@babel/preset-env']
+function generaseBaseConfig() {
```

Comment:
It's supposed to be "generate", right? 😁

Reply:
Fixed :)